tests/tests.js

```diff
@@ -1,14 +1,184 @@
 const tape = require('tape');
-const request = require('supertest');
+const supertest = require('supertest');
 const server = require('../server.js');
+const axios = require('axios');
+const jwt = require('jsonwebtoken');
+const nock = require('nock');
+const flush = require('./flush_database.js');
+require('dotenv').config();
+
+let id;
+axios
+  .post(`${process.env.URI}/places`, { en: { name: 'guesthouse mario' } })
+  .then(res => {
+    id = res.data._id;
+  });
 
 tape('home route test: GET request to /', t => {
-  request(server)
+  supertest(server)
     .get('/')
     .end((err, res) => {
-      if (err) throw err;
+      t.error(err, '/ home route does not return an error');
       t.equal(res.status, 200, 'should return status code 200');
-      t.ok(res.text.includes('<div id="list-page-content">'), 'should return home page containing test string "<div id="list-page-content">"');
+      t.ok(res.text.includes('<div id="list-page-content">'));
+      t.end();
+    });
+});
+
+tape('Test POST arabic event', t => {
```

Comment:
Not a blocker but ideally I'd like to see a bit of edge case handling. What happens when you send bad data or missing properties, etc.?

Reply:
not sure if i can test that since we are mocking the post request which is giving back a valid id.
so no matter what it's going to work.
also there is "required" on properties that the api must have in order for a successful POST request.

ps: #75

Reply:
Yes, but I'm guessing the OTP sends back some sort of error message when it recieves a bad request. You could nock the bad response and test that you have appropriate error handling for it.